### PR TITLE
Feature/testing routes fix

### PR DESCRIPTION
**Pull request by iiimba**
Testing routes implemented, comments fixed
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/22%23discussion_r58704077%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/22%23discussion_r58704300%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/22%23discussion_r58704661%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/22%23discussion_r59113984%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/22%23discussion_r59113996%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/22%23discussion_r59114007%22%2C%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/22%23discussion_r59118417%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%20c8abb3a12db725da8fe5b1883a762da4731939b9%20DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Controllers/AccountController.cs%2021%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/22%23discussion_r58704661%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22What%20the%20reason%20using%20Attribute%20Name%20%5C%22GetUserById%5C%22%3F%22%2C%20%22created_at%22%3A%20%222016-04-06T13%3A33%3A13Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6068068%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/TerraVenil%22%7D%7D%2C%20%7B%22body%22%3A%20%22It%20isn%27t%20I%20wrote%2C%20because%20I%20do%20not%20know%29%22%2C%20%22created_at%22%3A%20%222016-04-09T11%3A33%3A35Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15160198%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/iiimba%22%7D%7D%2C%20%7B%22body%22%3A%20%22It%20isn%27t%20I%20wrote%2C%20so%20I%20do%20not%20know%29%22%2C%20%22created_at%22%3A%20%222016-04-09T17%3A36%3A26Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15160198%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/iiimba%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Controllers/AccountController.cs%3AL39-48%22%7D%2C%20%22Pull%20c8abb3a12db725da8fe5b1883a762da4731939b9%20DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/TestRoutes/AccountTestRoutes.cs%2019%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/22%23discussion_r58704077%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22What%20about%20moving%20%5C%22/%5C%22%20inside%20RouteConstants.GetUser%20%3F%22%2C%20%22created_at%22%3A%20%222016-04-06T13%3A29%3A54Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6068068%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/TerraVenil%22%7D%7D%2C%20%7B%22body%22%3A%20%22Fixed%22%2C%20%22created_at%22%3A%20%222016-04-09T11%3A31%3A03Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15160198%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/iiimba%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/TestRoutes/AccountTestRoutes.cs%3AL1-73%22%7D%2C%20%22Pull%20c8abb3a12db725da8fe5b1883a762da4731939b9%20DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/TestRoutes/AccountTestRoutes.cs%2024%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ITsvetkoFF/Kv-013/pull/22%23discussion_r58704300%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22I%20don%27t%20see%20extensions...We%20already%20disscussed%20this...%22%2C%20%22created_at%22%3A%20%222016-04-06T13%3A31%3A17Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6068068%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/TerraVenil%22%7D%7D%2C%20%7B%22body%22%3A%20%22Added%20extension%20methods%22%2C%20%22created_at%22%3A%20%222016-04-09T11%3A32%3A33Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/15160198%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/iiimba%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/TestRoutes/AccountTestRoutes.cs%3AL1-73%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [x] <a href='#crh-comment-Pull c8abb3a12db725da8fe5b1883a762da4731939b9 DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/TestRoutes/AccountTestRoutes.cs 19'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-013/pull/22#discussion_r58704077'>File: DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/TestRoutes/AccountTestRoutes.cs:L1-73</a></b>
- <a href='https://github.com/TerraVenil'></a> What about moving "/" inside RouteConstants.GetUser ?
- <a href='https://github.com/iiimba'></a> Fixed
- [x] <a href='#crh-comment-Pull c8abb3a12db725da8fe5b1883a762da4731939b9 DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/TestRoutes/AccountTestRoutes.cs 24'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-013/pull/22#discussion_r58704300'>File: DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/TestRoutes/AccountTestRoutes.cs:L1-73</a></b>
- <a href='https://github.com/TerraVenil'></a> I don't see extensions...We already disscussed this...
- <a href='https://github.com/iiimba'></a> Added extension methods
- [ ] <a href='#crh-comment-Pull c8abb3a12db725da8fe5b1883a762da4731939b9 DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Controllers/AccountController.cs 21'></a> &nbsp;<b><a href='https://github.com/ITsvetkoFF/Kv-013/pull/22#discussion_r58704661'>File: DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Controllers/AccountController.cs:L39-48</a></b>
- <a href='https://github.com/TerraVenil'></a> What the reason using Attribute Name "GetUserById"?
- <a href='https://github.com/iiimba'></a> It isn't I wrote, because I do not know)
- <a href='https://github.com/iiimba'></a> It isn't I wrote, so I do not know)

<a href='https://www.codereviewhub.com/ITsvetkoFF/Kv-013/pull/22?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/ITsvetkoFF/Kv-013/pull/22?approve=1'></a>&nbsp;<a href='https://github.com/ITsvetkoFF/Kv-013/pull/22'></a>
<a href='#crh-end'></a>

### CODE REVIEW

File: DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/TestRoutes/AccountTestRoutes.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+namespace GitHubExtension.Security.Tests.TestRoutes
+{
+    public class AccountTestRoutes : TestRoutesConfig
+    {
+        [Fact]
+        public void AccountGetUserTest()
+        {
+            url += RouteConstants.ApiAccount +
+                "/" + Regex.Replace(
```
Comment (TerraVenil) [inline DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/TestRoutes/AccountTestRoutes.cs]: What about moving "/" inside RouteConstants.GetUser? I don't like this symbol duplication in each test...

File: DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/TestRoutes/AccountTestRoutes.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                RouteConstants.GetUser,
+                RouteConstants.Id_guid,
+                "644e1dd7-2a7f-18fb-b8ed-ed78c3f92c2b");
+
+            config.ShouldMap(url)
```
Comment (TerraVenil) [inline DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/TestRoutes/AccountTestRoutes.cs:27]: I don't see extensions...We already disscussed this...

File: DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Controllers/AccountController.cs [hunk 3]
```diff
@@ -38,8 +39,9 @@ public class AccountController : BaseApiController
             _userManager = userManager;
         }
 
-        [Authorize(Roles = "Admin")]
-        [Route("user/{id:guid}", Name = "GetUserById")]
+        [HttpGet]
+        [Authorize(Roles = RoleConstants.Admin)]
+        [Route(RouteConstants.GetUser, Name = "GetUserById")]
```
Comment (TerraVenil) [inline DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Controllers/AccountController.cs:44]: What the reason using Attribute Name "GetUserById"?

File: DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/TestRoutes/AccountTestRoutes.cs (hunk 1 above)
Comment (iiimba) [inline DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/TestRoutes/AccountTestRoutes.cs]: Fixed

File: DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/TestRoutes/AccountTestRoutes.cs (hunk 2 above)
Comment (iiimba) [inline DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/TestRoutes/AccountTestRoutes.cs:27]: Added extension methods

File: DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Controllers/AccountController.cs (hunk 3 above)
Comment (iiimba) [inline DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Controllers/AccountController.cs]: It isn't I wrote, because I do not know)

Comment (iiimba) [inline DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Controllers/AccountController.cs:44]: It isn't I wrote, so I do not know)

**Pull request merged**